WzComparerR2/CharaSimControl/GearGraphics.cs

```diff
@@ -530,12 +530,12 @@ public FormattedTextRenderer()
             Color defaultColor;
             Color orangeColor;
 
-            public void DrawString(Graphics g, string s, Font font, int x, int x1, ref int y, int height, Color? orangeColor = null)
+            public void DrawString(Graphics g, string s, Font font, int x, int x1, ref int y, int height, Color? orangeColor = null, Color? textColor = null)
             {
                 //初始化环境
                 this.g = g;
                 this.drawX = x;
-                this.defaultColor = Color.White;
+                this.defaultColor = (textColor == null ? Color.White : (Color)textColor);
```

Comment:
```suggestion
                this.defaultColor = textColor ?? Color.White;
```

코딩 컨벤션 유지를 위해 요청 드립니다.